Engines/Wine/Verbs/D9VK/script.js

```diff
@@ -28,6 +28,16 @@ class D9VK {
         return this;
     }
 
+    selectGithubVersion(wizard) {
+        const versions = fetchGithubReleases("Joshua-Ashton", "d9vk", wizard);
+
+        if (!this.d9vkVersion || typeof this.d9vkVersion !== 'string') {
+            return versions[0];
+        } else {
+            return versions.find(version => version.name === this.d9vkVersion);
+        }
+    }
```

Comment:
I'm a friend of such utility functions. The only issue I have with it is that we have it multiple times in our codebase. @plata do you have any idea how to improve on this?

Reply:
We could have something like `WizardUtils`. But honestly, I'm not sure if it makes too much sense. It would increase the complexity a bit for extracting these few lines of rather trivial code.

Reply:
I have deleted the methods because they don't seem to be necessary any more